saleor/graphql/account/mutations/staff.py

```diff
@@ -210,11 +212,21 @@ def clean_is_active(cls, is_active, instance, user):
         if not is_active:
             if user == instance:
                 raise ValidationError(
-                    {"is_active": "Cannot deactivate your own account."}
+                    {
+                        "is_active": ValidationError(
+                            "Cannot deactivate your own account.",
+                            code=AccountErrorCode.DEACTIVATE_OWN_ACCOUNT,
+                        )
+                    }
                 )
             elif instance.is_superuser:
                 raise ValidationError(
-                    {"is_active": "Cannot deactivate superuser's account."}
+                    {
+                        "is_active": ValidationError(
+                            "Cannot deactivate superuser's account.",
+                            code=AccountErrorCode.DEACTIVATE_SUPERUSER_ACCOUNT,
```

Comment:
🤔 Wouldn't that mean that most if not all error codes would start with CANNOT? I think I'd prefer to avoid negatives and use errors like `ACCOUNT_PROTECTED`. Ideally, the error code could be used to finish the following sentence: "the operation failed because...".